.github/workflows/examples-infineon.yaml

```diff
@@ -107,41 +107,82 @@ jobs:
                       out/artifacts/infineon-psoc6-light/chip-psoc6-lighting-example.out \
                       /tmp/bloat_reports/
 
-            - name: Build example CYW30739 Apps
+            - name: Build some CYW30739 Lighting Apps
               run: |
                   ./scripts/run_in_build_env.sh \
                     "./scripts/build/build_examples.py \
-                      --target cyw30739-cyw930739m2evb_01-light \
-                      --target cyw30739-cyw930739m2evb_01-lock \
-                      --target cyw30739-cyw930739m2evb_01-ota-requestor \
-                      --target cyw30739-cyw930739m2evb_01-switch \
+                      --target cyw30739-light \
                       build \
                       --copy-artifacts-to out/artifacts \
                     "
             - name: Get light size stats
               run: |
                   .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py \
-                    cyw30739 cyw930739m2evb_01 light \
-                    out/artifacts/cyw30739-cyw930739m2evb_01-light/chip-cyw30739-lighting-example.elf \
+                    cyw30739 CYW30739B2-P5-EVK-01 light \
+                    out/artifacts/cyw30739-light/lighting_app-CYW30739B2-P5-EVK-01.elf \
+                    /tmp/bloat_reports/
+                  .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py \
+                    cyw30739 CYW30739B2-P5-EVK-02 light \
+                    out/artifacts/cyw30739-light/lighting_app-CYW30739B2-P5-EVK-02.elf \
                     /tmp/bloat_reports/
-            - name: Get lock size stats
-              run: |
                   .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py \
-                    cyw30739 cyw930739m2evb_01 lock \
-                    out/artifacts/cyw30739-cyw930739m2evb_01-lock/chip-cyw30739-lock-example.elf \
+                    cyw30739 CYW30739B2-P5-EVK-03 light \
```

Comment:
does a single target generated several build outputs? why don't we have `cyw30739-p5-evk-03-light` target to build specific things? I find it surprising that a single build will generate so many elf files.

Could you explain the build process a bit to help me understand?

Reply:
Thank you for having reviewed it.
A brief explanation of the build process would be:
1. GN uses the `matter_wpan_sdk_board_list` variable in `examples/<app-name>/infineon/cyw30739/args.gni` to generate build rules for all supported boards into ninja files.
2. A `ninja -C <out>` command would build all supported boards.

In this change, I make the CYW30739 builder build all supported boards without specifying the board name. I expected CI to build all CYW30739-supported boards by default.

Reply:
updated.